backend/app/resources/InvestigatorEndpoint.py

```diff
@@ -19,6 +20,7 @@ def get(self, id):
         return self.schema.dump(model)
 
     def delete(self, id):
+        db.session.query(StudyInvestigator).filter_by(investigator_id=id).delete()
```

Comment:
I know you're planning on adding a backend test for this in your work on #402 - that will be a great addition.